wasmtime.yaml

```diff
@@ -23,7 +23,10 @@ pipeline:
     with:
       repository: https://github.com/bytecodealliance/wasmtime
       tag: v${{package.version}}
-      expected-commit: 8eefa236f8ef0cc766977e0c0cbb0d602132dfa4
+      expected-commit: 2e584e84127f0123c5a9211310140419e12082c2
+
+  #  # mitigate GHSA-h97m-ww89-6jmq
+  # - uses: rust/cargobump
```

Comment:
If we are not using cargobump This can be removed instead of leaving it as a comment.